### ISSUE TITLE
Runtime error,probably tensorflow ver newer have many error &my input file

### ISSUE THREAD

**Issue by buyvtuygJW** [skyil7/Piano2Midi#4]
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:From Piano2Midi.py:18: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From Piano2Midi.py:21: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.

#IN SHORT VER 2.1 NOT COMPATIBLE IF NEED NEW TF MANUALLY UPDATE THOSE IMPORT BOISSS.

**Comment by skyil7** [comment]
Hi, LJWxyz.
Well, There was A LOT OF changes and updates in TF 2.0.
So I think this repo won't work with TF 2.x version. You should `pip` lower version of TF.

**Comment by buyvtuygJW** [comment]
thanks.^^ For github is it possible to create multiple version of a project?

**Comment by buyvtuygJW** [comment]
sadly.TF==1.15,also does not completely work.have too much changes.
BTW where should we put the input file.^^ Thanks

**Comment by buyvtuygJW** [comment]
assume need to give full path

Input Your .WAV Filename:E:\Coding\PythonModulus\Others\Piano2Midi2020\01.kanate_vocal.wav
E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\magenta\music\audio_io.py:76: WavFileWarning: Chunk (non-data) not understood, skipping it.
  native_sr, y = scipy.io.wavfile.read(six.BytesIO(wav_data))

**ERROR:tensorflow:Error recorded from prediction_loop: 'NoneType' object has no attribute 'note_sequence'
WARNING:tensorflow:Reraising captured error
Traceback (most recent call last):
  File "Piano2Midi.py", line 99, in <module>
    infer(file)
  File "Piano2Midi.py", line 69, in infer
    predictions = list(estimator.predict(input_fn, yield_single_examples=False))
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 3078, in predict
    rendezvous.raise_errors()
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\tpu\error_handling.py", line 136, in raise_errors
    six.reraise(typ, value, traceback)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\six.py", line 696, in reraise
    raise value
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 3072, in predict
    yield_single_examples=yield_single_examples):
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 622, in predict
    features, None, ModeKeys.PREDICT, self.config)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 2857, in _call_model_fn
    config)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1149, in _call_model_fn
    model_fn_results = self._model_fn(features=features, **kwargs)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 3126, in _model_fn
    features, labels, is_export_mode=is_export_mode)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 1663, in call_without_tpu
    return self._call_model_fn(features, labels, is_export_mode=is_export_mode)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\tensorflow_estimator\python\estimator\tpu\tpu_estimator.py", line 1994, in _call_model_fn
    estimator_spec = self._model_fn(features=features, **kwargs)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\magenta\models\onsets_frames_transcription\train_util.py", line 81, in wrapped_model_fn
    return model_fn(features, labels, mode, params, config)
  File "E:\Coding\PYFastCache\PYVenv\Venv37_gpuacc\lib\site-packages\magenta\models\onsets_frames_transcription\model.py", line 405, in model_fn
    sequence_label=labels.note_sequence,
AttributeError: 'NoneType' object has no attribute 'note_sequence'**

#The TF==1.15 also too new!magenta too outdated.lol

**Comment by skyil7** [comment]
> sadly.TF==1.15,also does not completely work.have too much changes.
> BTW where should we put the input file.^^ Thanks

You don't have to place your file in designated directory.

Just input your directory with file name like below.
`files\input.wav`

**Comment by skyil7** [comment]
Sorry for late reply.

As you said, Magenta and this repo is too outdated.. I even can not remember how I code this. So.. I can not help you so much. sorry. T.T

**Comment by LJWxyz** [comment]
Ahaha,it's normal.I also create a game two years ago.no blank~~ahah .Thanks
for reply though

**Comment by buyvtuygJW** [comment]
Ahaha, it's fine .I also got create game two years ago. Now Blank.ahha
Thank you for your reply.^^
